src/services/api/wallet/useTransaction.ts

```diff
@@ -27,15 +52,23 @@ export const payInvoice = async ({
     .catch((err) => err);
 };
 
-export const useGetTransactions = (id: number) =>
-  useQuery<Transaction[], Error>({
-    queryFn: () => getTransaction(id),
-    queryKey: ["transaction", id],
-    refetchOnWindowFocus: false,
-    enabled: !!id,
+export const useWithdrawSats = () =>
+  useMutation({
+    mutationFn: withdrawSats,
   });
 
-export const usePayInvoice = () =>
+export const validateAddress = async ({
+  lnAddress,
+}: {
+  lnAddress: string;
+}): Promise<any> => {
+  return axiosBase
+    .post("/api/lightning/validate-address", { lnAddress })
+    .then((res) => res.data)
+    .catch((err) => err);
```

Comment:
Is your intention to use the error as a result? This would show up on the `onSuccess` callback in your mutation rather than `onError`

Reply:
yh , resembles the previous implementation